src/main/rx/sbus.c

```diff
@@ -142,19 +141,6 @@ static uint8_t sbusFrameStatus(rxRuntimeConfig_t *rxRuntimeConfig)
     }
     sbusFrameData->done = false;
 
-    DEBUG_SET(DEBUG_SBUS, DEBUG_SBUS_FRAME_FLAGS, sbusFrameData->frame.frame.channels.flags);
```

Comment:
What is the reason for removing the debug code here?

Reply:
Restored